waspc/src/Wasp/Util/ShellCommand.hs

```diff
@@ -0,0 +1,22 @@
+module Wasp.Util.ShellCommand where
+
+import Data.String (IsString (fromString))
+
+data ShellCommandArg = Raw String | Quoted String
+  deriving (Eq)
+
+instance Show ShellCommandArg where
+  show (Raw arg) = arg
+  show (Quoted arg) = "\"" ++ arg ++ "\""
+
+instance IsString ShellCommandArg where
+  fromString = Raw
+
+showShellArgs :: [ShellCommandArg] -> String
+showShellArgs = unwords . map show
+
+getShellArgValues :: [ShellCommandArg] -> [String]
```

Comment:
This one is a bit hard to understand. It basically unpacks the values. But I found it hard to understand from the name.

Maybe you could rather have `toRaw :: ShellCommandArg -> ShellCommandArg` and then you can do `(show . toRaw) <$> args` if you want to show them all without quotes -> because that is what you are using this function for. And then it is at least very clear this is what you are doing.